app/components/second/header/header-component/index.tsx

```diff
@@ -6,36 +6,39 @@ import {CloseOutlined} from "@ant-design/icons";
 import {Link} from "react-scroll";
 
 export const HeaderComponent = () => {
-    const [width, setWidth] = React.useState(769);
+    const [width, setWidth] = React.useState<number>();
```

Comment:
use only useState, not all React object. Remove React from the import if component works without it one